website/members/tests.py

```diff
@@ -92,11 +92,51 @@ def test_get_signup_person_page(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'members/signup_person_form.html')
 
+    def test_signup_person_page_linebreaks(self):
+        # test that it respectes the line breaks of the original description
+        Category.objects.create(name='Activo', description='foo\nbar\n', fee=50)
+        response = self.client.get(reverse('signup_person'))
+        self.assertIn(b'foo<br>bar', response.content)
+
+    def test_signup_person_page_categories(self):
+        # test that it show only human-related categories
+        cat = Category.objects.create(name=Category.BENEFACTOR_GOLD, description='', fee=500)
+        response = self.client.get(reverse('signup_person'))
+        self.assertNotIn(cat, response.context_data['categories'])
```

Comment:
Cat en este caso es un objecto... Como funciona el `in` en esos casos?

Reply:
No sé exactamente la teoría de qué hace el objeto de Django ahí... pero funciona, porque hice el test primero, fallaba, y luego excluí esa categoría y empezó a pasar ok. O sea que el "in" funcionaba bien (el objeto debe hashearse por algo que tenga sentido, etc)